gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/DagManagerTest.java

```diff
@@ -184,6 +184,43 @@ static Dag<JobExecutionPlan> buildDag(String id, Long flowExecutionId, String fl
     return new JobExecutionPlanDagFactory().createDag(jobExecutionPlans);
   }
 
+  // This creates a dag like this
+  //  D1  D2 D3
+  //    \ | /
+  //     DN4
+  //    / | \
+  //  D5 D6  D7
+  public static Dag<JobExecutionPlan> buildDagWithMultipleNodesAtDifferentLevels(String id, Long flowExecutionId, String flowFailureOption,
```

Comment:
since this isn't a test but a utility method that seems unused in this defining class, does it really belong here?